components/ui/button.tsx

```diff
@@ -9,15 +9,18 @@ const buttonVariants = cva(
   {
     variants: {
       variant: {
-        default: "bg-primary text-primary-foreground hover:bg-primary/90",
-        destructive:
-          "bg-destructive text-destructive-foreground hover:bg-destructive/90",
+        // Built-in variants
+        // default: "bg-primary text-primary-foreground hover:bg-primary/90",
         outline:
```

Comment:
Looks like the default outline should be a light gray border. Then, the selected item is with an orange border.
![image](https://github.com/LetsGetTechnical/gridiron-survivor/assets/362527/b76d0c7a-431f-43b5-8ee9-a821527a5d57)